assessment_images_bucket.tf

```
@@ -0,0 +1,83 @@
+# ------------------------------------------------------------------------------
+# Create S3 buckets to store assessment images.
+# ------------------------------------------------------------------------------
+
+resource "aws_s3_bucket" "production" {
+  acl    = "private"
+  bucket = local.production_bucket_name
+
+  # Until this policy attachment happens, we don't have permission
+  # to provision the bucket.
+  depends_on = [
+    aws_iam_role_policy_attachment.provision_bucket_production
+  ]
+
+  provider = aws.images_production
```

We've traditionally kept the keys that are generic to _all_ Terraform resource types in a separate section above the keys that are specific to the resource type.  This suggested change shows what I mean:
```suggestion
  provider = aws.images_production
  # Until this policy attachment happens, we don't have permission
  # to provision the bucket.
  depends_on = [
    aws_iam_role_policy_attachment.provision_bucket_production
  ]

  acl    = "private"
  bucket = local.production_bucket_name

```

I'm open to alphabetizing the first section (traditionally we have ordered them by perceived importance) but I think it makes sense to keep the two types of tags separate.  As always, I am open to discussion if the group feels this is wrong-headed.

FWIW, you seem to have followed the scheme I outlined above throughout the rest of this PR.

---

Oops! Thanks for catching that. I have updated both resources in ca57fd3.